### PR TITLE
Fix removing pure python security proxies

### PR DESCRIPTION
**Pull request by jinty**
The test is simple but requires zope.security to be installed.
I chose that over writing the test in zope.security

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ python:
     - pypy
 install:
     - pip install .
+    - pip install zope.security
```
Comment (mgedmin) [inline .travis.yml]: You can do it in one command `pip install .[testing]`

**Comment by mgedmin** [comment]
Would be nice to have a changelog entry for this bugfix.

Otherwise, LGTM.

**Comment by tseaver** [comment]
@jinty Thanks for the patch!

**Comment by agroszer** [comment]
My hunch is that this broke winbot:

http://winbot.zope.org/builders/zope.proxy_py_270_32/builds/1956/steps/test/logs/stdio

**Comment by tseaver** [comment]
@agroszer Thanks for pointing that out.  I think #14 should resolve that issue.

**Comment by jamadden** [comment]
Sorry, I probably don't understand, but why can't winbot test with zope.security? Why is this a special case? Windows is hard 😢

**Comment by tseaver** [comment]
@jamadden  Winbot tests are buildout-driven, rather than using `tox`.  This PR didn't adjust the `buildout.cfg` file to include `zope.security`, which meant that the test failed due to an import error.  #14 adjusts the default `[test]` part to include `zope.security`, as well as making the test relying on it optional, and adds a `[test-wo-zope-security]` part to exercise that.

**Pull request merged**